scripts/sync.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "fs";
+
+/**
+ * This script synchronizes the in-repo ecosystem schema JSON to the latest
+ * version.
+ */
+
+async function updateSchema() {
+    console.log("Updating games.json...")
+    const response = await fetch("https://thunderstore.io/api/experimental/schema/dev/latest/");
+    if (response.status !== 200) {
```

Comment:
Since this is run before all builds and publish command, Is totally preventing the execution what we want if the schema file is unavailable?

Reply:
The build failing was intentional (as the idea was you should always have the latest version of the schema), but I'm going to remove it from the build script after all since we don't do any validation on it, meaning you could end up with an invalid schema in your build in some specific failure scenarios.